content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst

```diff
@@ -73,567 +79,669 @@ Electronic Invoice Credentials
-  and the payment date, it’s possible to create a credit/debit note to decrease/increase the
-  amount to pay in ARS.
+When creating a :guilabel:`Credit/Debit` note related to a FCE document:
 
-In the workflow we can have two scenarios:
+- use the :guilabel:`Credit and Debit Note` buttons, so all the information from the invoice is
+  transferred to the new :guilabel:`Credit and Debit Note`;
+- the document letter should be the same as than the originator document (either A or B);
+- the same currency as the source document must be used. When using a secondary currency there is
+  an exchange difference if the currency rate is different between the emission day and the payment
+  date. It is possible to create a credit/debit note to decrease/increase the amount to pay in ARS.
 
-#. The FCE is rejected so the Credit Note should have the field “FCE, is Cancellation?” as True.
-#. The Credit Note, is created with the negative amount to annulate the FCE document,
-   in this case the field “FCE, is Cancellation?” must be empty (false).
+.. image:: argentina/credit-debit-notes-button.png
+   :align: center
+   :alt: Credit & debit notes buttons.
 
-.. image:: argentina/argentina_edi_12.png
+When creating a :guilabel:`Credit Note` we can have two scenarios:
+
+#. the FCE is rejected so the :guilabel:`Credit Note` should have the field :guilabel:`FCE,
+   is Cancellation?` as *True*; or;
+#. the :guilabel:`Credit Note`, is created to annulate the FCE document, in this case the field
+   :guilabel:`FCE, is Cancellation?` must be *empty* (false).
+
+.. image:: argentina/fce-es-cancelation.png
    :align: center
+   :alt: FCE: Es Cancelación?
 
 Invoice printed report
 ~~~~~~~~~~~~~~~~~~~~~~
-The PDF report related to electronic invoices that have been validated by the AFIP includes
-a barcode at the bottom of the format which represent the CAE number, the Expiration Date is
-also displayed as it’s legal requirement:
 
-.. image:: argentina/argentina_edi_14.png
-   :align: center
+The :guilabel:`PDF Report` related to electronic invoices that have been validated by the AFIP
+includes a barcode at the bottom of the format which represents the CAE number. The expiration date
+is also displayed as it is a legal requirement.
 
+.. image:: argentina/invoice-printed-report.png
+   :align: center
+   :alt: Invoice printed report.
 
-Troubleshooting and Auditing
+Troubleshooting and auditing
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-For auditing and troubleshooting purposes you can get the detailed information of an
-invoice number that has been  previously sent to the AFIP,
 
-.. image:: argentina/argentina_edi_23.png
-   :align: center
+For auditing and troubleshooting purposes, it is possible to obtain detailed information of an
+invoice number that has been previously sent to the AFIP. To retrieve this information, go into
+:doc:`Developer Mode <../../../../general/developer_mode>`, then go to the
```

Comment:
Either:
- use `doc` and start from the root to limit the number of `../`: `/applications/general/developer_mode`
- use `ref` (exceptionally) with the target `developer-mode`.